web/src/components/tests/SolutionInput.test.tsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import {render, screen} from "@testing-library/react";
+import {SolutionEntry} from "../Question";
+
+describe("Solution input tests", () => {
+    it("can enter maths and we get a good latex result", async () => {
+        const {rerender} = render(<SolutionEntry latex={""} setLatex={jest.fn()}/>)
+        await screen.findByRole("textbox")
+        // It's unfortunate we have to control this with the internal state
+        rerender(<SolutionEntry latex={"\\frac{1}{2}"} setLatex={jest.fn()}/>)
```

Comment:
Oh I see what you mean. And I'm not sure what we'd do when we remove the "output:" text 🤔

Reply:
Yeah - the page level tests would be way nicer where we can assert the actual behaviour we want when someone submits something. These component level tests really aren't that useful here :/